src/main/java/seedu/address/logic/commands/Command.java

```diff
@@ -8,6 +9,8 @@
  */
 public abstract class Command {
 
+    public static final Index TAB_ID = Index.fromOneBased(3);
```

Comment:
Maybe we can have an enum for TAB_ID instead to make it more clear.

Reply:
I was also hesitating between enum and Index, in the end I chose Index because it's easier to convert it to an int and then use it as input to the MainWindow UI class, avoiding a switch statement which may add unnecessary clutter.